src/core/eme/index.ts

```diff
@@ -79,32 +89,36 @@ function clearEMESession(mediaElement : HTMLMediaElement) : Observable<never> {
  * appropriate one supported by the user's browser.
  * @param {HTMLMediaElement} mediaElement
  * @param {Array.<Object>} keySystems
- * @param {Subject} errorStream
  * @returns {Observable}
  */
 export default function EMEManager(
   mediaElement : HTMLMediaElement,
-  keySystemsConfigs: IKeySystemOption[],
-  errorStream: Subject<Error|ICustomError>
-) : Observable<never> {
+  keySystemsConfigs: IKeySystemOption[]
+) : Observable<IEMEManagerEvent> {
   if (__DEV__) {
     keySystemsConfigs.forEach((config) => assertInterface(config, {
       getLicense: "function",
       type: "string",
     }, "keySystem"));
   }
+  const warning$ = new Subject<Error|ICustomError>();
+  const warningEvents$ : Observable<IEMEWarningEvent> = warning$
+    .pipe(map(err => ({
+      type: "warning" as "warning",
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast. ([typescript:S4325](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4325))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=283&issues=AWUoCSifOTG07Bqb98zH&open=AWUoCSifOTG07Bqb98zH)

[ISSUE_KEY]: AWUoCSifOTG07Bqb98zH